lib/apiservers/service/restapi/handlers/vch_create.go

```diff
@@ -413,10 +414,17 @@ func buildCreate(ctx context.Context, d *data.Data, finder *find.Finder, vch *mo
 				c.Proxies = fromImageFetchProxy(vch.Registry.ImageFetchProxy)
 				_, _, err := c.Proxies.ProcessProxies()
 				if err != nil {
-					return nil, util.NewError(400, fmt.Sprintf("Error processing proxies: %s", err))
+					return nil, util.NewError(http.StatusBadRequest, fmt.Sprintf("Error processing proxies: %s", err))
 				}
 			}
 		}
+
+		if len(vch.SyslogAddr) > 0 {
```

Comment:
Other places in this handler, we seem to be checking `!= ""`. I don't know which pattern is better, but I think it would be good to be consistent.

Reply:
`!= ""` makes more sense.
I was still in the java mindset that strings cannot be directly compared with equals